### PR TITLE
NH-20727 Provide uninstall

### PR DESCRIPTION
**Pull request by jskiba**
* fixed test for debian:stretch
* fixed ansible-linter
* added uninstallation scripts

### CODE REVIEW

**Review by dominik-maslyk-swi — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -112,131 +148,153 @@ jobs:
...
-            - test-install-debian-bullseye
-            - test-install-centos-7
-            - test-install-fedora-32
-            - test-install-fedora-36
-            - test-install-kali
-            - test-install-oracle-8
-            - test-install-oracle-9
+            - test-install-and-uninstall-ubuntu-focal
+            - test-install-and-uninstall-ubuntu-focal-vm
+            - test-install-and-uninstall-ubuntu-jammy
```
Comment (poleszcz) [inline .circleci/config.yml:288]: Don't you want to try also 23.04 - Lobster ?

File: .circleci/config.yml [hunk 2]
```diff
@@ -112,131 +148,153 @@ jobs:
...
             - ansible-lint
-            - test-install-ubuntu-focal
-            - test-install-ubuntu-focal-vm
-            - test-install-ubuntu-jammy
-            - test-install-debian-stretch
-            - test-install-debian-buster
-            - test-install-debian-bullseye
-            - test-install-centos-7
-            - test-install-fedora-32
-            - test-install-fedora-36
```
Comment (poleszcz) [inline .circleci/config.yml:236]: There is already Fedora 38. Maybe add it or replace 36 ?

File: .circleci/config.yml [hunk 3]
```diff
@@ -112,131 +148,153 @@ jobs:
...
           filters: *filters-test-jobs
       - release-to-ansible-galaxy:
           requires:
             - ansible-lint
-            - test-install-ubuntu-focal
-            - test-install-ubuntu-focal-vm
-            - test-install-ubuntu-jammy
-            - test-install-debian-stretch
-            - test-install-debian-buster
-            - test-install-debian-bullseye
```
Comment (poleszcz) [inline .circleci/config.yml:233]: There is Debian 12 Bookworm ..... add ?

**Comment by poleszcz** [comment]
@jskiba All comments regarding new OS releases can be ignored or discussed with PO. It's relatively low cost to just try running tests with newest releases ... or it can be deffered under some other JIRA item.

The rest ... seems ok. As long as it works in tests you can ship it.

**Comment by jskiba** [comment]
> @jskiba All comments regarding new OS releases can be ignored or discussed with PO. It's relatively low cost to just try running tests with newest releases ... or it can be deffered under some other JIRA item.
> 
> The rest ... seems ok. As long as it works in tests you can ship it.

@poleszcz created Jira ticket for that https://swicloud.atlassian.net/browse/NH-58387. We will do it later

**Review by poleszcz — APPROVED** [review]

**Pull request merged**